cli/src/main/scala/scalaxb/compiler/wsdl11/Driver.scala

```diff
@@ -254,7 +254,7 @@ class Driver extends Module { driver =>
       case (VersionPattern(1, 0 | 1, _), style)                    =>
         // Same as 0.13.x
         generateDispatchFromResource(style, "/httpclients_dispatch0130", config)
-      case (VersionPattern(2, 0 | 2, _), style)                    =>
+      case (VersionPattern(2, 0, _), style)                    =>
```

Comment:
```suggestion
      case (VersionPattern(2, _, _), style)                    =>
```

I suggest wildcard match here so it matches all 2.x.